docs/source/plugins/06_solver_hooks.rst

```diff
@@ -219,7 +219,7 @@ three-phase flow it is required a plugin that handles the oil-water system.
 
 .. note ::
```

Comment:
@ggrbill  I didn't get this note. Could you help me, pls?

Reply:
This is tricky to explain.

I think we could change the name of things. Like the `Two-phase (Gas-Liquid) System of Mechanistic Model` Could be just `Two-Phase Mechanistic Model. 

Then, this note could be:
"
Note that the two-phase (Gas-Liquid) Mechanistic Model can be used for both two and three-phase flows (Fluid FLow Type selected by the user in ALFAsim). When no Liquid-Liquid System is implemented by the plugin, the three-phase flow is adapted to work with the traditional two-phase Mechanistic Model, in which all liquid phases are considered as only one.
"

See if you understand it. Perhaps it needs a little bit more improvement.

Reply:
I see that this is hard to explain in this context since it requires a deeper knowledge of the hydrodynamic modeling. But I think that your suggestion is much better than the original text. Thanks!